pkg/apis/events/v1alpha1/cloudauditlogssource_types.go

```diff
@@ -107,26 +107,31 @@ func (*CloudAuditLogsSource) GetGroupVersionKind() schema.GroupVersionKind {
 	return SchemeGroupVersion.WithKind("CloudAuditLogsSource")
 }
 
-///Methods for pubsubable interface
+// Methods for identifiable interface.
 
-// PubSubSpec returns the PubSubSpec portion of the Spec.
-func (s *CloudAuditLogsSource) PubSubSpec() *duckv1alpha1.PubSubSpec {
-	return &s.Spec.PubSubSpec
+func (s *CloudAuditLogsSource) IdentitySpec() *duckv1alpha1.IdentitySpec {
+	return &s.Spec.IdentitySpec
 }
 
-// PubSubStatus returns the PubSubStatus portion of the Status.
-func (s *CloudAuditLogsSource) PubSubStatus() *duckv1alpha1.PubSubStatus {
-	return &s.Status.PubSubStatus
+func (s *CloudAuditLogsSource) IdentityStatus() *duckv1alpha1.IdentityStatus {
```

Comment:
maybe add comments as with the PubSub counterparts..

// IdentityStatus returns the IdentityStatus portion of the Status? Same for Spec, and in every other place?